src/main.js

```diff
@@ -1 +1,9 @@
 console.log('Hello World');
+
+function printNumber(number) {
+    for (let i = 1; i <= number; i++) {
```

Comment:
Try to understand the different between `var` & `let` in this case. What would be different if it was `var` instead of `let`?

https://stackoverflow.com/questions/40519064/i-am-confused-with-javascript-let-and-var-in-for-loop